GameData/KSPCommunityFixes/Settings.cfg

```diff
@@ -302,6 +302,7 @@ KSP_COMMUNITY_FIXES
   // ##########################
 
   // Reduce scene loading time by caching the current save in memory instead of loading it from disk
+  // Additionally forces all scene transitions to happen synchronously.
```

Comment:
This should be a separate cfg option - being able to toggle these independently is vital for identifying when one of them causes an issue.